um_canonicalize/tags.yaml

```diff
@@ -308,12 +351,21 @@ categories:
   Tense:
     - PRS
     - PST
+    - PST.FREQ
+    - INF
```

Comment:
Is this "infinitive"? Finitivity is a separate category (Sylak-Glassman 5.10): there is `FIN` and `NFIN`.